wdl/subworkflows/get_preseq_metrics.wdl

```diff
@@ -0,0 +1,46 @@
+version 1.0
+
+
+import "../tasks/stampipes.wdl"
+import "../tasks/preseq.wdl"
+
+
+workflow get_preseq_metrics {
+    input {
+        File nuclear_bam
+        PreseqLcExtrapParams params = object {
+            defects: false,
```

Comment:
I was not sure where to fix this, but I think this is the way until we know if/how often this actually needs to be `true`.